internal/services/sentinel/sentinel_alert_rule_scheduled_resource_test.go

```diff
@@ -339,6 +361,32 @@ QUERY
 `, r.template(data), data.RandomInteger)
 }
 
+func (r SentinelAlertRuleScheduledResource) upgradeVersion(data acceptance.TestData, update bool) string {
+	version := "1.0.4"
+	if update {
+		version = "1.0.5"
+	}
+	return fmt.Sprintf(`
```

Comment:
Is there a reason not to just use a string here and avoid complication of the bool to string evaluation, and be able to provide values in the calling test? e.g.
```suggestion
func (r SentinelAlertRuleScheduledResource) upgradeVersion(data acceptance.TestData, version string) string {
	return fmt.Sprintf(`
```